### PR TITLE
Update translation strings after UNAIDS review

### PR DESCRIPTION
**Pull request by r-ash**
Leaving this as a draft for the time being as there are a couple of things I want to clarify with Ian here
* ~~NAVIGATOR_ART_IS_SPECTRUM_DESC has not been translated~~ - Has now
* ~~Portuguese has a mix of PLHIV and PVVIH - which one is correct?~~ - PVVIH, updated this
* ~~"Untreated" in "UNTREATED_PLHIV_ATTEND" is that correct?~~ - No, updated now

### CODE REVIEW

File: inst/traduire/fr-translation.json [hunk 1]
```diff
@@ -114,8 +114,8 @@
     "STATUS_CALIBRATION_STRATA_LABEL": "État des strates d’étalonnage connues: ",
     "INFECTIONS_CALIBRATION_LEVEL_LABEL": "Nouveau niveau d’étalonnage des infections:  ",
     "INFECTIONS_CALIBRATION_STRATA_LABEL": "Nouvelles strates d’étalonnage des infections: ",
-    "PLHIV_CALIBRATION_LEVEL_LABEL": "Niveau d’étalonnage des PVVIH: ",
-    "PLHIV_CALIBRATION_STRATA_LABEL": "Strates d’étalonnage des PVVIH:  ",
+    "PLHIV_CALIBRATION_LEVEL_LABEL": "Niveau de calibration PVVIH:  ",
+    "PLHIV_CALIBRATION_STRATA_LABEL": "Catégorie d'étalonnage PVVIH: ",
     "POPULATION_CALIBRATION_LEVEL_LABEL": "Étalonnage de la population: ",
```
Comment (r-ash) [inline inst/traduire/fr-translation.json:119]: @rtesra there is a mix of spaces and tabs after the `:` in these strings. Is that important?

File: inst/report/comparison_report.Rmd [hunk 2]
```diff
@@ -165,13 +165,13 @@ if (print_en) {
...
 if (print_fr) {
-  cat(paste0("## ", country, " Subnational HIV Estimates \n"))
-  cat(paste0("Naomi estimates for **", quarter_t2, "** (report generated ", report_date, ")"))
+  cat(paste0("## ", country, " Estimativas subnacionais do VIH \n"))
+  cat(paste0("Les estimations de Naomi pour **", quarter_t2, "** (rapport généré ", report_date, ")"))
 }
 
 if (print_pt) {
-  cat(paste0("## ", country, " Subnational HIV Estimates \n"))
-  cat(paste0("Naomi estimates for **", quarter_t2, "** (report generated ", report_date, ")"))
```
Comment (r-ash) [inline inst/report/comparison_report.Rmd:174]: We didn't get a string for these from UNAIDS, we should send this to them to check

File: inst/report/comparison_report.Rmd [hunk 3]
```diff
@@ -183,23 +183,37 @@ if (print_pt) {
...
 if (print_fr) {
-  cat("### Comparison of Naomi model estimates to data inputs  \n")
-  cat("\n### Household Survey \n")
+  cat("### Comparaison des estimations du modèle Naomi avec les entrées de données\n")
+  cat("\n### Données d’enquête auprès des ménages \n")
 }
 
 if (print_pt) {
-  cat("### Comparison of Noami model estimates to data inputs  \n")
-  cat("\n### Household Survey \n")
```
Comment (r-ash) [inline inst/report/comparison_report.Rmd:202]: Same as above, I don't think we asked UNAIDS about this?

File: inst/report/comparison_report.Rmd [hunk 4]
```diff
@@ -213,18 +227,18 @@ Cross-sectional estimates for HIV prevalence, ART coverage, and HIV incidence ar
...
 if (print_fr) {
-  cat("This report compares Naomi estimates for ", quarter_t2, "to the following population-baes survey(s): \n")
-  cat("\n### Household Survey \n")
+  cat("Ce rapport compare les estimations de Naomi pour ", quarter_t2, "à l'enquête de population suivante(s): \n")
+  cat("\n### Données d’enquête auprès des ménages \n")
 }
 
 if (print_pt) {
-  cat("This report compares Naomi estimates for ", quarter_t2, "to the following population-baes survey(s): \n")
-  cat("\n### Household Survey \n")
```
Comment (r-ash) [inline inst/report/comparison_report.Rmd:227]: This one too we didn't ask UNAIDS. I also updated to `population-based` that is correct? That was a typo?

File: inst/report/comparison_report.Rmd [hunk 5]
```diff
@@ -250,18 +264,18 @@ We anticipate that the Naomi estimates will be closely aligned with the survey e
...
 
 ::: {#translate lang="pt"}
-We anticipate that the Naomi estimates will be closely aligned with the survey estimates at a national level. Some variation is expected between model estimates and survey estimates at lower Admin levels where survey estimates may be sparse for certain age and sex groups.
+Esperamos que as estimativas de Naomi estejam estreitamente alinhadas com as estimativas de pesquisas em nível nacional. Espera-se alguma variação entre as estimativas do modelo e as estimativas da pesquisa em níveis administrativos mais baixos, onde as estimativas da pesquisa podem ser esparsas para alguns grupos de idade e sexo.
 :::
 
 ```{r, echo = FALSE, results = 'asis'}
 
 if (print_en) {cat("\n### Geographic distribution \n")}
-if (print_fr) {cat("\n### Geographic distribution \n")}
```
Comment (r-ash) [inline inst/report/comparison_report.Rmd:263]: Check with UNAIDS

File: inst/report/comparison_report.Rmd [hunk 6]
```diff
@@ -352,8 +366,8 @@ if(is_empty(survey_prev2)) {
 ```{r, echo = FALSE, results = 'asis'}
 
 if (print_en) {cat("\n### Age distribution \n")}
-if (print_fr) {cat("\n### Age distribution \n")}
```
Comment (r-ash) [inline inst/report/comparison_report.Rmd:355]: Check with UNAIDS

File: inst/report/comparison_report.Rmd [hunk 7]
```diff
@@ -382,11 +396,11 @@ fig2 <- age_bar_plotly (data,
 :::
 
 ::: {#translate lang="fr"}
-*Static images of individual plots can be downloaded by hovering on the right hand corner*
```
Comment (r-ash) [inline inst/report/comparison_report.Rmd:385]: Check with UNAIDS

File: inst/report/comparison_report.Rmd [hunk 8]
```diff
@@ -424,6 +438,10 @@ programme_data <- c(art_year, anc_prev, anc_art)
 if (sum(is.na(programme_data)) <3 ) {
 
   cat(paste0("\n### Données du programme opérationnel: \n", sep = "\n"))
+  
+  cat(paste0("Pour les composantes de prévalence du VIH et de couverture du TAR du modèle, les chiffres de prestation de services de TAR informent les estimations infranationales du nombre de PVVIH. Étant donné que la taille de l'échantillon de l'enquête auprès des ménages dans chaque district est relativement petite, les données régulièrement déclarées sur la prévalence du VIH chez les femmes enceintes assistant à leur première consultation prénatale, extraites du système national d'information sur la santé, sont utilisées pour améliorer les estimations de la répartition spatiale du VIH. \n", sep = "\n"))
+
+  cat(paste0("\nCe rapport compare les estimations de Naomi à: \n", sep = "\n"))
```
Comment (r-ash) [inline inst/report/comparison_report.Rmd:444]: Check this one with UNAIDS

File: inst/report/comparison_report.Rmd [hunk 9]
```diff
@@ -446,6 +466,10 @@ programme_data <- c(art_year, anc_prev, anc_art)
 if (sum(is.na(programme_data)) <3 ) {
 
   cat(paste0("\n### Dados do programa operacional: \n", sep = "\n"))
+  
+  cat(paste0("Para os componentes de prevalência do VIH e cobertura de TARV do modelo, os números de prestação de serviços de TARV informam as estimativas subnacionais para o número de PVVIH. Uma vez que o tamanho da amostra do inquérito domiciliar em cada distrito é relativamente pequeno, os dados relatados rotineiramente sobre a prevalência do HIV entre mulheres grávidas na primeira consulta pré-natal, extraídos do sistema nacional de informações de saúde, são usados para melhorar as estimativas do padrão espacial do VIH. \n", sep = "\n"))
+
+  cat(paste0("\nEste relatório compara as estimativas da Naomi com: \n", sep = "\n"))
```
Comment (r-ash) [inline inst/report/comparison_report.Rmd:472]: Check

File: inst/report/comparison_report.Rmd (hunk 2 above)
Comment (rtesra) [reply inst/report/comparison_report.Rmd:174]: These come from the summary report - have updated.

File: inst/report/comparison_report.Rmd (hunk 3 above)
Comment (rtesra) [reply inst/report/comparison_report.Rmd:202]: Yeah - sorry these got left off I'll follow up.

File: inst/report/comparison_report.Rmd (hunk 4 above)
Comment (rtesra) [reply inst/report/comparison_report.Rmd:227]: Will follow up with UNAIDS

File: inst/report/comparison_report.Rmd (hunk 6 above)
Comment (rtesra) [reply inst/report/comparison_report.Rmd:355]: Copied over from summary report.

**Review by rtesra — APPROVED** [review]

**Pull request merged**